src/controllers/company.details.controller.ts

```diff
@@ -73,6 +80,20 @@ export const confirmCompanyStartRequest = async (req: Request, res: Response, ne
   }
 };
 
+const isExtensionDueDateWithinLimit = (company: ExtensionsCompanyProfile): boolean => {
+  const dueDateString: string = company.accountsDue;
+  if (!dueDateString || dueDateString.length === 0) {
```

Comment:
should check if string is empty as well, could have dueDateString = " "

Reply:
I though so but we it would mean making that field nullable in the Company profile model - do we want to make that sort of change? If we don't change it compiler will complain if we leave that field out of the dummy data.

Reply:
sorry, i was wrong, length will obviously capture whitespace, was thinking of something else.

Reply:
i've contradicted myself again!  could we add a trim() in front of the .length(). , that should do it

Reply:
trim added.